generator.js

```diff
@@ -5,15 +5,15 @@ const os = require("os");
 
 require("./scripts/installPackages.js"); // install packages
 
-// serve data
-let command = "node ./node_modules/docsify-cli/bin/docsify serve ./docs --port 9505";
+const wrapQuotes = s => `"${s}"`;
+
+const DOCSIFY_PATH = "./node_modules/docsify-cli/bin/docsify";
 
-if (os.platform() === "win32") {
-    command = "node \"./node_modules/docsify-cli/bin/docsify\" serve ./docs --port 9505";
-}
+// serve data
+const command = `node ${os.platform() === "win32" ? wrapQuotes(DOCSIFY_PATH) : DOCSIFY_PATH} serve ./docs --port 9505`;
```

Comment:
I've been wondering why we're even checking for win32 here. Regardless of the operating system currently being used, a path wrapped in quotes should still be able to work (testing with `cd "../"` on Linux; unsure if it would work with this specific command but I firmly believe it should). Same case the other way around.

Would it not be better to just remove the platform check entirely?

Reply:
I totally agree, I'll fix it now